docker/Dockerfile

```diff
@@ -191,5 +191,9 @@ ENV LD_LIBRARY_PATH=${INSTALL_DIR}/lib
 RUN python3 -m pip install --upgrade pip setuptools wheel \
   && python3 -m pip install --no-cache-dir -r requirements3.txt
 
+RUN echo '#!/bin/bash\npython3 -m pip "$@"' > /usr/bin/pip3 \
```

Comment:
I'm surprised the binary isn't installed, maybe it lives in a different path somewhere? That wrapper works too of course, if nothing else helps ...

Reply:
It isn't installed because I explicitly uninstalled the default python environment. We need to use the vcpkg compiled python and it's proved tricky to have both environments so I'd rather have just the one we need to prevent any collisions in the future.